DESCRIPTION

```diff
@@ -159,7 +159,9 @@ Suggests:
     tgp,
     TH.data,
     wavelets,
-    xgboost (>= 0.6-2)
+    xgboost (>= 0.6-2),
+    sp,
+    gridExtra
```

Comment:
We already use `cowplot` for gridded plot arrangements. Its more powerful in combination with ggplot. Give it a try :)

Reply:
I'll check that. And as gridExtra was required for the vignette, I'll remove it from the suggest section (see my comment related to sp)